AutomatedTesting/Gem/PythonTests/scripting/ScriptCanvas_TwoComponents_InteractSuccessfully.py

```diff
@@ -55,15 +55,13 @@ def ScriptCanvas_TwoComponents_InteractSuccessfully():
     import azlmbr.math as math
     import azlmbr.asset as asset
     import azlmbr.bus as bus
+    import azlmbr.editor as editor
```

Comment:
Minor: unused import?

Reply:
It's used here: editor.EditorComponentAPIBus(bus.Broadcast, 'BuildComponentPropertyList', component)